runtime/src/main/kotlin/kotlin/native/ForeignException.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package kotlin.native
+
+import kotlin.RuntimeException
+
+public class ForeignException internal constructor(val objCException: Any?)  : RuntimeException()
```

Comment:
If this class becomes Obj-C-specific, it should be named accordingly.

Reply:
The class itself if platform-agnostic. Compiler implementation currently supports NSException only but this could be extended in future. I revert field name to nativeException as it may hold generic payload.

Reply:
Class is supposed to be platform-agnostic. Currently implementation of compiler part creates ForeignException wrapper for NSException only and terminates on other exceptions but this may be extended in future. Renamed field to nativeExveption to make naming consistent.